flows/codelingo/review/Gopkg.lock

```diff
@@ -35,7 +35,7 @@
   version = "1.3"
```

Comment:
I know this PR didn't introduce this, but I'm wondering why we have a nested Gopkg.lock